docs/standard/library.md

```diff
@@ -1,105 +1,124 @@
 ---
-title: .NET Standard Library
-description: .NET Standard Library
-keywords: .NET, .NET Core
+title: .NET Standard | Microsoft Docs
+description: Learn what is .NET Standard, its versions and the .NET platforms supported.
+keywords: .NET Standard, PCL, .NET
 author: richlander
 ms.author: mairaw
-ms.date: 06/20/2016
+ms.date: 03/16/2017
 ms.topic: article
 ms.prod: .net
 ms.technology: dotnet-standard
 ms.devlang: dotnet
 ms.assetid: c044882c-af15-45f2-96d1-534557a5ee9b
 ---
 
-# .NET Standard Library
```

Comment:
@svick you are correct which is why the issue the issue becomes confusing.  In the case of c a library became a standard because it was the most common one.  I mentioned the term was a misnomer in this context because .net now has several libraries none of which  is any more or less standard than the other.  In this context I think it is helpful to think of the standard as a common attribute which is a different entity from the libraries that implement it (an interface for those who think of it technically).